src/components/menu/menu-item/menu-item.style.js

```diff
@@ -0,0 +1,243 @@
+import styled, { css } from 'styled-components';
+import { StyledSubmenu, StyledSubmenuTitle, StyledSubmenuBlock } from '../submenu-block/submenu.style';
+import { baseTheme } from '../../../style/themes';
+
+const StyledMenuItemWrapper = styled.div`
+  display: inline-block;
+  font-size: 13px;
+  font-weight: 700;
+  height: 40px;
+  padding: 0px 16px;
+  position: relative;
+  cursor: pointer;
+
+  .carbon-portrait {
+    top: -1px;
+  }
+
+  &:focus {
+    z-index: 1;
+  }
+
+  ${({ menuType, theme }) => menuType === 'primary' && css`
+    background-color: ${theme.colors.white};
+    color: ${theme.colors.slate}; 
+
+    a, a:hover, a:focus{
+      color: ${theme.colors.slate};
+      text-decoration: none;
+      background-color: transparent;
+    }
+
+    [data-component='icon'] {
+      color: ${theme.colors.slate};
+    }
+
+    &:after {
+      transition: all 200ms;
+      content: "";
+      position: absolute;
+      height: 0;
+      background-color: transparent;
+      bottom: 0;
+      left: 0%;
+      right: 0%;
+    }
+
+    &:hover:after {
+      background-color: ${theme.colors.primary}; 
+      height: 3px;
+      left: 0% !important;
+      right: 0% !important;
+    }
+
+    ${({ selected }) => selected && css`
+      &:after{
+        left: 10px;
+        right: 10px;
+        background-color: #00DC00; 
+        height: 3px;
+      }
+    `}
+    
+  `}
+
+  ${({ menuType, theme }) => menuType === 'secondary' && css`
+    background-color: ${theme.colors.slate};
+    color: ${theme.colors.white};
+
+    a, a:hover, a:focus {
+      text-decoration: none;
+      color: ${theme.colors.white};
+      background-color: transparent;
+    }
+
+    [data-component='icon'] {
+      color: ${theme.colors.white};
+    }
+
+    :hover &, :hover {
+      background-color: ${theme.colors.primary}; 
+    }
+  `}
+
+  ${({ hasSubmenu, menuType, theme }) => hasSubmenu && css`
+    padding: 0;
+
+    ${StyledSubmenuTitle}{
+      ${StyledMenuItemWrapper}{
+        padding-right: 32px;
+      }
+    }
+
+    :before {
+      margin-top: -2px;
+      pointer-events: none;
+      position: absolute;
+      right: 16px;
+      top: 50%;
+      z-index: 2;
+      
+      ${menuType === 'primary' && css`
+          content: "";
+          width: 0;
+          height: 0;
+          border-top: 5px solid ${theme.colors.slate};
+          border-right: 4px solid transparent;
+          border-bottom: 4px solid transparent;
+          border-left: 4px solid transparent;
+      `}
+
+      ${menuType === 'secondary' && css`
+          content: "";
+          width: 0;
+          height: 0;
+          border-top: 5px solid ${theme.colors.white};
+          border-right: 4px solid transparent;
+          border-bottom: 4px solid transparent;
+          border-left: 4px solid transparent;
```

Comment:
if it differs only in border-top color, then you could just do:
`border-top: 5px solid ${menuType === 'primary' ? theme.colors. slate : theme.colors.white };`